toolkit/tools/pkg/imagecustomizerlib/customizepartitions_test.go

```diff
@@ -59,7 +59,7 @@ func TestCustomizeImagePartitions(t *testing.T) {
 		},
 	}
 
-	imageConnection, err := connectToImage(buildDir, outImageFilePath, mountPoints)
+	imageConnection, err := connectToImage(buildDir, outImageFilePath, false, mountPoints)
```

Comment:
nit: I find writing what the `false` or `true` is next to it improves readability (i.e. `false /*<flag-name>*/`.

Reply:
Fixed.